src/tests/components/ForecastSummary.test.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import ForecastSummary from "../../components/ForecastSummary";
+
+describe("ForecastSummary", () => {
+  const validProps = {
+    date: 1525046400000,
+    description: "Stub description",
+    icon: "800",
+    temperature: {
+      min: 12,
+      max: 22,
+    },
+    onSelect: () => {},
+  };
+  it("renders correctly", () => {
+    const { asFragment } = render(
+      <ForecastSummary
+        date={validProps.date}
+        description={validProps.description}
+        icon={validProps.icon}
+        temperature={validProps.temperature}
+        onSelect={validProps.onSelect}
+      />
+    );
+
+    expect(asFragment()).toMatchSnapshot();
+  });
+  it("renders correct values for props", () => {
+    const { getByText, getByTestId } = render(
+      <ForecastSummary
+        date={validProps.date}
+        description={validProps.description}
+        icon={validProps.icon}
+        temperature={validProps.temperature}
```

Comment:
Theres an error in the console for this test caused by not passing in the required `onSelect` prop, should add this in